tests/toitp/summary-toitp-test.toit

```diff
@@ -8,7 +8,8 @@ import .utils
 main args:
   out := run-toitp args []
   required-output-snippets := [
-    "snapshot: ",
+    "Uuid: ",
```

Comment:
Why are size and uuid capitalized in the output?

Reply:
changed to lower case.
done.